dist/vespa.spec

```diff
@@ -162,6 +162,7 @@ Requires: gdb
 Requires: nc
 Requires: net-tools
 Requires: unzip
+Requires: zstd # For inspection of compressed jdisc access log files
```

Comment:
I think we already have vespa_zstd installed in /opt/vespa-dependencies/bin

Reply:
Thanks!